synapse/config/logger.py

```diff
@@ -202,6 +205,15 @@ def sighup(signum, stack):
     if getattr(signal, "SIGHUP"):
         signal.signal(signal.SIGHUP, sighup)
 
+    # make sure that the first thing we log is a thing we can grep backwards
+    # for
+    logging.warn("***** STARTING SERVER *****")
+    logging.warn(
+        "Server %s version %s",
+        sys.argv[0], get_version_string(synapse),
+    )
+    logging.info("Server hostname: %s", config.server_name)
```

Comment:
any reason why this is INFO and the other two are WARN?

Reply:
only in that if somebody has really turned their logs down, we probably want the WARNs but can live without this one.